tests/training/metrics/span_based_f1_measure_test.py

```diff
@@ -163,3 +172,65 @@ def test_span_f1_can_build_from_params(self):
+
+        # Make prediction so that it is exactly correct.
+        prediction_tensor[:, 0, 1] = 1
+        prediction_tensor[:, 1, 0] = 1
+        prediction_tensor[:, 2, 10] = 1
+        prediction_tensor[:, 3, 5] = 1
+        prediction_tensor[:, 4, 11] = 1
+        prediction_tensor[:, 5, 0] = 1
+
+        metric = SpanBasedF1Measure(self.vocab, "tags")
+        metric(prediction_tensor, gold_tensor, mask)
+        metric_dict = metric.get_metric()
+
+        # We merged the continued ARG1 label into a single span, so there should
+        # be exactly 1 true positive for ARG1 and nothing present for C-ARG1
+        assert metric._true_positives["ARG1"] == 1
+        # The labels containing continuation references get merged into
+        # the labels that they continue, so they should never appear in
+        # the precision/recall counts.
+        assert "C-ARG1" not in metric._true_positives.keys()
+        assert metric._true_positives["V"] == 1
+        assert metric._true_positives["ARGM-ADJ"] == 1
+
+        numpy.testing.assert_almost_equal(metric_dict["recall-ARG1"], 1.0)
+        numpy.testing.assert_almost_equal(metric_dict["precision-ARG1"], 1.0)
+        numpy.testing.assert_almost_equal(metric_dict["f1-measure-ARG1"], 1.0)
+        numpy.testing.assert_almost_equal(metric_dict["recall-V"], 1.0)
+        numpy.testing.assert_almost_equal(metric_dict["precision-V"], 1.0)
+        numpy.testing.assert_almost_equal(metric_dict["f1-measure-V"], 1.0)
+        numpy.testing.assert_almost_equal(metric_dict["recall-ARGM-ADJ"], 1.0)
+        numpy.testing.assert_almost_equal(metric_dict["precision-ARGM-ADJ"], 1.0)
+        numpy.testing.assert_almost_equal(metric_dict["f1-measure-ARGM-ADJ"], 1.0)
+        numpy.testing.assert_almost_equal(metric_dict["recall-overall"], 1.0)
+        numpy.testing.assert_almost_equal(metric_dict["precision-overall"], 1.0)
+        numpy.testing.assert_almost_equal(metric_dict["f1-measure-overall"], 1.0)
+
+        # This doesn't directly feature in the test, but it demonstrates that continued arguments
+        # are represented as single spans in the output of the official perl script. If you run
+        # pytest with the -v flag, you will be able to see the perl script output.
+        gold_file_path = os.path.join(self.TEST_DIR, "gold_conll_eval.txt")
+        prediction_file_path = os.path.join(self.TEST_DIR, "prediction_conll_eval.txt")
+        with open(gold_file_path, "a+") as gold_file, open(prediction_file_path, "a+") as prediction_file:
+            # Use the same bio tags as prediction vs gold to make it obvious by looking
+            # at the perl script output if something is wrong.
+            write_to_conll_eval_file(gold_file, prediction_file, 4, sentence, bio_tags, bio_tags)
+
+        perl_script_command = ["perl", "./scripts/srl-eval.pl", prediction_file_path, gold_file_path]
+        exit_code = subprocess.check_call(perl_script_command)
```

Comment:
Is it easy to parse the result of this?  I know it's easy in python to get the stderr / stdout; is the output format easy to read a number off of?  That'd make this a more useful test.